subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/catalog/CatalogPluginsGroovyDSLIntegrationTest.groovy

```diff
@@ -187,4 +187,84 @@ dependencyResolutionManagement {
         alias << ['greeter', 'some.greeter', 'some-greeter']
     }
 
+    def "can apply a plugin alias that has sub-accessors"() {
+        String pluginVersion = '1.5'
+        String firstLevelTask = 'greet'
+        String firstLevelPluginId = 'com.acme.greeter'
+        String secondLevelPluginTask = 'greet-second'
+        String secondLevelPluginId = 'com.acme.greeter.second'
+        new PluginBuilder(file("greeter"))
+            .addPluginWithPrintlnTask(firstLevelTask, 'Hello from first plugin!', firstLevelPluginId, "FirstPlugin")
+            .addPluginWithPrintlnTask(secondLevelPluginTask, 'Hello from second plugin!', secondLevelPluginId, "SecondPlugin")
+            .publishAs("some", "artifact", pluginVersion, pluginPortal, executer)
+            .allowAll()
+
+        file("settings.gradle") << """
+dependencyResolutionManagement {
+    versionCatalogs {
+        libs {
+            alias('greeter').toPluginId('$firstLevelPluginId').version('$pluginVersion')
+            alias('greeter-second').toPluginId('$secondLevelPluginId').version('$pluginVersion')
+        }
+    }
+}"""
+        withPluginAliases(["libs.plugins.greeter", "libs.plugins.greeter.second"])
+
+        when:
+        succeeds(firstLevelTask, secondLevelPluginTask)
+
+        then:
+        outputContains 'Hello from first plugin!'
+        outputContains 'Hello from second plugin!'
+    }
+
+    def "can apply a plugin via buildscript that has sub-accessors"() {
```

Comment:
Looking some more into the added tests, I feel like this one is a superset of "can apply a plugin declared in a catalog via buildscript".
So maybe that other one can be removed. Wdyt?

Reply:
Good point. I will delete the other one